src/components/IssueSearchAdvanced.vue

```diff
@@ -71,7 +84,7 @@ const optionsFor = (field: string) =>
     major_incident_state: flawIncidentStates,
     affects__affectedness: affectAffectedness,
   })[field] || null;
-const shouldShowAdvanced = ref(route.query.mode === 'advanced');
+const shouldShowAdvanced = ref(true);
```

Comment:
If the `mode` query parameter is not used anymore, I'll remove it, is used in multiple files https://github.com/search?q=repo%3ARedHatProductSecurity%2Fosim+%22%3A+%27advanced%27%22&type=code

Reply:
I am not sure if that's something is/will be needed, may be @tboyetteredhat of @superbuggy can confirm

Reply:
We are using mode to show advanced search section.

Reply:
@tboyetteredhat The only usage I see is this one that @C-Valen removed, where is used apart from this file?

Reply:
If is not read anywhere I think we could remove it